salt/journal/critical-css.sls

```diff
@@ -0,0 +1,70 @@
+api-dummy-journal-repository:
+    cmd.run:
+        - name: echo "journal-repository is ready for api-dummy-repository"
+        - require:
+            - journal-repository
+        - require_in:
+            - cmd: api-dummy-repository
+
+api-dummy-nginx-vhost:
+    file.managed:
+        - name: /etc/nginx/sites-enabled/api-dummy.conf
+        - source: salt://journal/config/etc-nginx-sites-enabled-api-dummy.conf
+        - listen_in:
+            - service: nginx-server-service
+
+journal-local-demo-separate-folder:
+    file.directory:
+        - name: /srv/journal-local-demo
+        - user: {{ pillar.elife.deploy_user.username }}
+        - group: {{ pillar.elife.deploy_user.username }}
+
+    cmd.run:
+        - name: |
+            rsync -a --exclude='.git' --exclude 'app/config/parameters.yml' --exclude 'node_modules' --exclude 'var/*' --include 'var/.gitkeep' --delete /srv/journal/ /srv/journal-local-demo
+        - user: {{ pillar.elife.deploy_user.username }}
+        - require:
+            - file: journal-local-demo-separate-folder
+            - running-gulp
+
+journal-local-demo-parameters:
+    file.managed:
+        - name: /srv/journal-local-demo/app/config/parameters.yml
+        - source: salt://journal/config/srv-journal-local-demo-app-config-parameters.yml
+        - template: jinja
+        - user: {{ pillar.elife.deploy_user.username }}
+        - group: {{ pillar.elife.deploy_user.username }}
+        - require:
+            - journal-local-demo-separate-folder
+
+journal-local-demo-cache-clean:
+    cmd.run:
+        - name: bin/console cache:clear --env=demo
+        - cwd: /srv/journal-local-demo
+        - user: {{ pillar.elife.deploy_user.username }}
+        - require:
+            - journal-local-demo-parameters
+
+journal-local-demo-nginx-vhost:
+    file.managed:
+        - name: /etc/nginx/sites-enabled/journal-local-demo.conf
+        - source: salt://journal/config/etc-nginx-sites-enabled-journal-local-demo.conf
+        - template: jinja
+        - require:
+            - nginx-config
```

Comment:
this has no `require` on `journal-local-demo-parameters`, which could trigger the recursion problem. Should be ok